extensions/eclipse-che-theia-remote-impl-che-server/src/node/che-server-devfile-service-impl.ts

```diff
@@ -553,13 +553,25 @@ export class CheServerDevfileServiceImpl implements DevfileService {
   }
 
   devfileV1toDevfileV2(devfileV1: cheApi.workspace.devfile.Devfile): Devfile {
+    const components: DevfileComponent[] = [];
+    if (devfileV1.components) {
+      devfileV1.components.forEach(component => {
+        if (component.type === 'cheEditor' || component.type === 'kubernetes') {
```

Comment:
I don't see why we filter out these components ?

Reply:
Giving them to the client does not have any sense, as it uses devfile2.

Reply:
Do you prefer to sent them to the client and mark with special attribute?